app/controllers/auth/AuthAction.scala

```diff
@@ -51,14 +51,14 @@ class AuthActionImpl @Inject()(
       res.replaceParams("redirect_uri", configDecorator.pertaxFrontendBackLink).toString()
     }
 
-  object GT100 {
+  object LT200 {
```

Comment:
Need to update the spec L100 no longer exists so needs to be removed from the code base and changed to 50 or 200 dependent on the scenario